PlayFabClientSDK/source/PlayFabHttp/PlayFabSysHttp.cs

```diff
@@ -5,27 +5,16 @@
 using System.Net.Http;
 using System.Text;
 using System.Threading.Tasks;
-using PlayFab.Json;
 
 namespace PlayFab.Internal
 {
     public class PlayFabSysHttp : IPlayFabHttp
     {
         private readonly HttpClient _client = new HttpClient();
-        private ISerializerPlugin _serializer;
-
-        public PlayFabSysHttp()
-        {
-            _serializer = (ISerializerPlugin)PluginManager.Instance.GetPlugin(PluginContract.Serializer);
-            if (_serializer == null)
-            {
-                _serializer = new PlayFabSerializer();
-                PluginManager.Instance.SetPlugin(PluginContract.Serializer, _serializer);
-            }
-        }
 
         public async Task<object> DoPost(string urlPath, PlayFabRequestCommon request, string authType, string authKey, Dictionary<string, string> extraHeaders)
```

Comment:
Remove string authType, string authKey.
They'll already have been atdded to extraHeaders before we get here.